src/engraving/utests/utils/scorerw.cpp

```diff
@@ -73,6 +73,7 @@ MasterScore* ScoreRW::readScore(const String& name, bool isAbsolutePath, ImportF
         score = nullptr;
     } else {
         for (Score* s : score->scoreList()) {
+            s->setIsOpen(true);
```

Comment:
We should not open scores implicitly here

Reply:
Yeah, I was trying to get the unit tests to succeed. I need to find another solution (and for some reason I can't run them on my machine, so I need to push the commit to the PR and wait for them)

Reply:
Which OS are you on? The unit tests only really work on Linux right now. If you're on Windows you could try building the tests within WSL2, though I wouldn't recommend trying to build/run the GUI that way.

Reply:
Ah, good to know that, thanks! I'm on windows now, but I do have the linux environment set up on my old laptop, I could try to run the tests there